example-configmap.yaml

```diff
@@ -13,19 +13,15 @@ data:
         <link href="styles.css" rel="stylesheet" type="text/css">
       </head>
       <body>
-        Hello from KOTS.
+        This is an example KOTS application.
         <p>This is text from a user config value: '{{repl ConfigOption "example_default_value"}}' </p>
```

Comment:
Any reason this line has paragraph tags but others don't?

Reply:
no reason. This is some very rough HTML and I'm not fussing about style. Open to iterations on this though.